ttsfm/models.py

```
@@ -0,0 +1,241 @@
+    speed: Optional[float] = None
+    max_length: int = 4096
+    validate_length: bool = True
+    
+    def __post_init__(self):
+        """Validate and normalize fields after initialization."""
+        # Ensure voice is a valid Voice enum
+        if isinstance(self.voice, str):
+            try:
+                self.voice = Voice(self.voice.lower())
+            except ValueError:
+                raise ValueError(f"Invalid voice: {self.voice}. Must be one of {list(Voice)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)